### ISSUE TITLE
Release more often?

### ISSUE THREAD

**Issue by Dialga** [civetweb/civetweb#626]
Do you think you can bundle a release for this project more often? Currently there seems to be only a release once or twice a year, which is quickly outdated. Or even perhaps provide daily binary builds on a server? Perhaps adopt the Release early, release often RERO time based philosophy? Just some ideas

**Comment by bel2125** [comment]
Yes, currently there is a release once or twice a year.
One of the limits are the manual tests that are performed on a variety of platforms - although I can probably drop some older platforms by now (like Windows XP and Debian 6 ARM 32-bit), and some of the (former) contributors who assisted in the manual tests have their projects already deployed and don't do releases that often. Some of the tests can be automated, but not all of them can run on "public infrastructure" like Travis CI or AppVeyor.

**Comment by laoshaw** [comment]
yes some old platforms really should be dropped to move civetweb forward at a faster speed, for example adding a template(e.g. mustache in C) support etc, there are quite a few C framework these days and so far civetweb still is in the lead
